src/server/api/controllers/users.controller.js

```diff
@@ -5,26 +5,16 @@ const knex = require('../../config/db');
 const Error = require('../lib/utils/http-error');
 const moment = require('moment-timezone');
 
-const getUsers = async () => {
-  try {
-    return await knex('users').select('users.id', 'users.title');
-  } catch (error) {
-    return error.message;
-  }
-};
+const getUsers = () => knex('users').select('*');
```

Comment:
This should not be part of the pr

Reply:
Yes I am aware of that and I will remove the the PR 82ac734 from here and just keep the other stuff but I believe I can do it until I am able to pull the merges from the users-controller-fix branch.